src/main/java/com/example/jwt/service/UserServiceImp.java

```diff
@@ -0,0 +1,8 @@
+package com.example.jwt.service;
+
+import com.example.jwt.entity.User;
+
+public interface UserServiceImp {
+
+    User RegisterUser(User user);
+}
```

Comment:
Impl Postfix is for the Implementation class not for the interface.

Reply:
I'll fix it today, thanks.